src/platform-integration/web/wasm.md

```diff
@@ -154,12 +123,16 @@ Server started on port 8080
 
 #### Load it in a browser
 
+The build command above builds the app to both JavaScript and Wasm.
+When the app is loaded, it will detect if the browser supports Wasm,
+and fall back to the JavaScript version if it doesn't.
+
 As of {{page.last-update}},
 [only **Chromium-based browsers**](#chrome-119-or-later)
 (Version 119 or later) are able to run Flutter/Wasm content. 
 
-If your configured browser meets the requirements, open
-[`localhost:8080`](http://localhost:8080) in the browser to view the app.
+Open [`localhost:8080`](http://localhost:8080) 
```

Comment:
I suspect that the broken build is due to this link.

Reply:
That link was introduced by a different PR, so should probably be fixed separately from this.